src/microbe_stage/BackgroundPlane.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using Godot;
+
+/// <summary>
+///   Manages the microbe background plane, optionally applies blur.
+/// </summary>
+public partial class BackgroundPlane : Node3D, IGodotEarlyNodeResolve
+{
+    private readonly StringName blurAmountParameter = new("blurAmount");
+    private readonly StringName textureAlbedoParameter = new("textureAlbedo");
+    private readonly StringName worldPositionParameter = new("worldPos");
+    private readonly StringName lightLevelParameter = new("lightLevel");
+    private readonly StringName distortionStrengthParameter = new("distortionFactor");
+
+    [Export]
+    private NodePath? blurPlanePath;
```

Comment:
With Godot4 we switched to the style where nodepaths aren't used anymore but direct node setting (there is still a lot of old code not updated but I enforce that proper usage in new code).